umap/static/umap/js/umap.controls.js

```diff
@@ -1478,14 +1478,17 @@ L.U.IframeExporter = L.Evented.extend({
       delete this.queryString.datalayers
     }
     const currentView = this.options.currentView ? window.location.hash : ''
-    return `${this.baseUrl}?${L.Util.buildQueryString(this.queryString)}${currentView}`
+    const queryString = L.extend({}, this.queryString, options)
+    return `${this.baseUrl}?${L.Util.buildQueryString(queryString)}${currentView}`
   },
 
   build: function () {
     const iframeUrl = this.buildUrl()
+    let fullUrl
     let code = `<iframe width="${this.dimensions.width}" height="${this.dimensions.height}" frameborder="0" allowfullscreen allow="geolocation" src="${iframeUrl}"></iframe>`
     if (this.options.includeFullScreenLink) {
-      code += `<p><a href="${this.baseUrl}">${L._('See full screen')}</a></p>`
+      fullUrl = this.buildUrl({ scrollWheelZoom: true })
```

Comment:
Why don't you define that variable directly on that line?

```suggestion
      const fullUrl = this.buildUrl({ scrollWheelZoom: true })
```

Reply:
:+1: